h/static/scripts/build-thread.js

```diff
@@ -238,6 +238,11 @@ var defaultOpts = {
    */
   filterFn: undefined,
   /**
+   * Function that returns true if an annotation of a certain
```

Comment:
`threadFilterFn` is a generic predicate function, it doesn't have to be used for filtering annotations by type. So the comment here could be adjusted to something like "A filter function which should return true if a given annotation and its replies should be displayed`"